lib/tower_error_tracker/reporter.ex

```diff
@@ -19,6 +19,13 @@ defmodule TowerErrorTracker.Reporter do
     :ok
   end
 
+  def report_event(%Tower.Event{kind: :message, reason: reason, level: level} = event) do
+    message = "[#{level}] #{reason}"
+    ErrorTracker.report({"message #{:erlang.phash2(message)}", message}, [], context(event))
```

Comment:
3rd attempt, trying to disambiguate messages with source location and function instead of message phash https://github.com/mimiquate/tower_error_tracker/pull/47

Reply:
yeah i was just coming back to report on this. you're way ahead of me.